wkcuber/utils.py

```diff
@@ -62,14 +62,14 @@ def add_verbose_flag(parser):
     parser.set_defaults(verbose=True)
 
 
-def add_anisotropic_flag(parser):
+def add_isotropic_flag(parser):
     parser.add_argument(
-        "--anisotropic",
-        "-a",
-        help="Activates Anisotropic downsampling. It will detect which dimension is the smallest and the largest. "
-        "The largest dimension will only be downsampled by 2 if it would be smaller or equal to the smallest "
-        "dimension in the next downsampling step.",
-        dest="anisotropic",
+        "--isotropic",
+        "-iso",
+        help="Activates isotropic downsampling. The default is anisotropic downsampling. "
+        "Therefore it will always downsample each dimension with the factor 2.",
```

Comment:
`Isotropic downsampling will always...`. Otherwise, it might be a bit unclear, since the previous sentence refers to anisotropic downsampling.